dvc/tree/git.py

```diff
@@ -74,9 +74,10 @@ def dvcignore(self):
         return cls(self, root)
 
     def open(
-        self, path, mode="r", encoding="utf-8"
+        self, path, mode="r", encoding=None
     ):  # pylint: disable=arguments-differ
         assert mode in {"r", "rb"}
+        encoding = encoding or "utf-8"
```

Comment:
Hmm, maybe `api.open` should use `utf-8` by default?

Reply:
@pared Ideally - no, it should use your default system encoding. This is how things like pathlib's Path do things. I believe, Saugat just changed this for a better practice for optional args and to match BaseTree.open (there are still some discrepancies in tree methods, that I left untouched when adding these trees).

Reply:
Yeah, it should use default system encoding, I just tried to make `api.open()` work without changing any behavior here.
Something to fix it later.